### PR TITLE
Add Powerline fonts

### PR DESCRIPTION
**Pull request by malyn**

### CODE REVIEW

File: pkgs/data/fonts/powerline-fonts/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "powerline-fonts-39c99c";
+
+  src = fetchgit {
```
Comment (edwtjo) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Please use fetchFromGitHub

File: pkgs/data/fonts/powerline-fonts/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      cp -v */*.ttf $out/share/fonts/truetype
+    '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/powerline/fonts;
+    description = "Patched fonts for Powerline users.";
+    longDescription = ''
+      Pre-patched and adjusted fonts for usage with the Powerline plugin.
+    '';
+    license = with licenses; [ asl20 free ofl ];
```
Comment (edwtjo) [inline pkgs/data/fonts/powerline-fonts/default.nix]: :) does this work?!

Comment (malyn) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Which part?  The multiple licenses thing?  Yes, apparently so (I found one or two other packages that did the same thing...), although it came up in PR #7493 as well and so I want to make a change here before we close this PR.  Any thoughts on how to handle these situations?

Comment (edwtjo) [inline pkgs/data/fonts/powerline-fonts/default.nix]: I say; use a space delimited string with the representations for the referenced licenses for now.

Comment (malyn) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Works for me.  Are you thinking of something like:

```
license = "asl20 free ofl";
```

Or the actual human-readable names of the licenses?

Comment (edwtjo) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Well, there is often a spdxId in `lib/licenses.nix` use that, if there isn't one update that file with an spdxId.

Comment (malyn) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Okay, will do.

Comment (edwtjo) [inline pkgs/data/fonts/powerline-fonts/default.nix]: No.. I'm talking out of my ass. Just use the shortname instead.

Comment (malyn) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Okay. :)

**Comment by malyn** [comment]
I fixed both issues.

File: pkgs/data/fonts/powerline-fonts/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "powerline-fonts-39c99c";
```
Comment (edwtjo) [inline pkgs/data/fonts/powerline-fonts/default.nix]: sorry, just noticed this. you should add a relevant date version, like 2015-04-24. the rationale I believe is that seeing the package name should give you an idea on how old the package is, a short hash doesn't allow that

Comment (malyn) [inline pkgs/data/fonts/powerline-fonts/default.nix]: Yep, I like that.  Fixed.

**Pull request merged**